src/database/populate.py

```
@@ -57,34 +57,41 @@ def _preprocess_csv(self) -> pd.DataFrame:
         :return: A Pandas DataFrame containing cleaned movie data.
         """
         data = pd.read_csv(self._csv_file_path)
-        data = data.drop_duplicates(subset=['names', 'date_x'], keep='first')
-
-        for col in ['crew', 'genre', 'country', 'orig_lang', 'status']:
-            data[col] = data[col].fillna('Unknown').astype(str)
-
-        data['crew'] = (
-            data['crew']
-            .str.replace(r'\s+', '', regex=True)
-            .apply(lambda x: ','.join(sorted(set(x.split(',')))) if x != 'Unknown' else x)
+        data = data.drop_duplicates(subset=["names", "date_x"], keep="first")
+
+        for col in ["crew", "genre", "country", "orig_lang", "status"]:
+            data[col] = data[col].fillna("Unknown").astype(str)
+
+        data["crew"] = (
+            data["crew"]
+            .str.replace(r"\s+", "", regex=True)
+            .apply(
+                lambda x: (
+                    ",".join(sorted(set(x.split(","))))
+                    if x != "Unknown"
+                    else x
+                )
+            )
         )
 
-        data['genre'] = data['genre'].str.replace('\u00A0', '', regex=True)
-        data['date_x'] = data['date_x'].astype(str).str.strip()
-        data['date_x'] = pd.to_datetime(data['date_x'], format='%Y-%m-%d', errors='raise')
-        data['date_x'] = data['date_x'].dt.date
-        data['orig_lang'] = data['orig_lang'].str.replace(r'\s+', '', regex=True)
-        data['status'] = data['status'].str.strip()
+        data["genre"] = data["genre"].str.replace("\u00a0", "", regex=True)
+        data["date_x"] = data["date_x"].astype(str).str.strip()
+        data["date_x"] = pd.to_datetime(
+            data["date_x"], format="%Y-%m-%d", errors="raise"
```

`pd.to_datetime(..., errors='raise')` will raise and abort the whole seed if any row has an unparsable date. If your CSV might contain dirty dates, consider `errors='coerce'` and then filter/log rows with `NaT`, or catch parsing exceptions and decide whether to skip or fix problematic rows. Failing fast might be OK for development, but tests / large CSV imports often need more tolerant handling.